integr8ly/tutorial-web-app-operator
# INTLY-10570 Update to Go Mod

Update from gopkg to go.mod

Verification: 
1. Build operator 
2. Deploy to cluster
3. Verify walkthroughs are available by accessing them through the route. 

If you get an error with the pod `unable to open database file` set the env `DATABASE_LOCATION` to `/tmp`  This is a known issue and not an issue in production.
